tasks/ebpf.py

```diff
@@ -0,0 +1,125 @@
+try:
+    from termcolor import colored
+except ImportError:
+    colored = None
+import contextlib
+import json
+import sys
+
+from invoke import task
+
+try:
+    from tabulate import tabulate
+except ImportError:
+    tabulate = None
+
+from .system_probe import build_object_files, is_root
+
+headers = [
+    "Program",
+    "Stack Usage",
+    "Instructions Processed",
+    "Instructions Processed limit",
+    "Verification time",
+    "Max States per Instruction",
+    "Peak States",
+    "Total States",
+]
+
+
+def tabulate_stats(stats):
+    table = list()
+    for key, value in stats.items():
+        row = list()
+        row.append(key)
+        row.append(value["stack_usage"])
+        row.append(value["instruction_processed"])
+        row.append(value["limit"])
+        row.append(value["verification_time"])
+        row.append(value["max_states_per_insn"])
+        row.append(value["peak_states"])
+        row.append(value["total_states"])
```

Comment:
💬 **suggestion**: ‏For loop would be more readable and shorter, wdyt?

This list of string seemed to be reused in the task as well !